infra/build/Makefile

```diff
@@ -16,7 +16,7 @@ SHELL := /usr/bin/env bash # Make will use bash instead of sh
 TERRAFORM_VERSION := 0.13.4
 CLOUD_SDK_VERSION := 313.0.1
 GSUITE_PROVIDER_VERSION := 0.1.22
-TERRAFORM_DOCS_VERSION := 0.6.0
+TERRAFORM_DOCS_VERSION := 0.10.1
```

Comment:
We will also need to bump https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/1967f1842600a982b2425f114388e76db6403c88/infra/build/Makefile#L30